### DISCUSSION TITLE
How do I use this?

### DISCUSSION THREAD

**Discussion by ghost** [waylaidwanderer/node-chatgpt-api#301] [Help]
I'm a newbie at Artificial Intelligence and have only been looking at it for the past week or so. I'm wondering how all of this works and how I can start jailbreaking Bing AI. Could someone please help me?

**Comment by waylaidwanderer** [comment]
Read the readme and then check out https://github.com/waylaidwanderer/node-chatgpt-api/blob/main/demos/use-bing-client.js
